build.sh

```diff
@@ -9,7 +9,9 @@ mono ./.nuget/NuGet.exe restore ./ScriptCs.sln
 
 # script
 mkdir -p artifacts/Release/bin
-xbuild ./ScriptCs.sln /property:Configuration=Release /nologo /verbosity:normal
-cp src/*/bin/Release/* artifacts/Release/bin/
-mono ./packages/xunit.runners.1.9.2/tools/xunit.console.clr4.exe test/ScriptCs.Tests.Acceptance/bin/Release/ScriptCs.Tests.Acceptance.dll /xml artifacts/ScriptCs.Tests.Acceptance.dll.TestResult.xml /html artifacts/ScriptCs.Tests.Acceptance.dll.TestResult.html
+xbuild ./build/Build.proj /property:Configuration=Release /nologo /v:m /target:UpdateProjectVersions
+xbuild ./ScriptCs.sln /property:Configuration=Release /nologo /v:m /fl "/flp:LogFile=artifacts/msbuild.log;Verbosity=diagnostic;DetailedSummary"
+xbuild ./build/Build.proj /property:Configuration=Release /nologo /v:m /target:RestoreCommonVersionInfo
 
+cp src/*/bin/Release/* artifacts/Release/bin/ 2>/dev/null
```

Comment:
It redirects `STDERR` (file descriptor 2) to the black hole that is `/dev/null`. Basically anything written to `STDERR` is thrown away. `STDOUT` has a file descriptor of 1.